pages/1.12/storage/persistent-volume/index.md

```diff
@@ -479,9 +479,8 @@ response:
 }
 ```
 
-**Note:** A running task will show `stagedAt`, `startedAt`, and `version` in addition to the information provided above.
-
-You can then
+**Note:** 
+A running task will show `stagedAt`, `startedAt`, and `version` in addition to the information provided above. You can do the following actions:
```

Comment:
I do not think `You can do the following actions:` should be part of the `Note`.